src/Core/src/Core/IVisualTreeHelper.cs

```diff
@@ -0,0 +1,12 @@
+﻿using System;
+using System.Collections.Generic;
+using System.ComponentModel;
+
+namespace Microsoft.Maui
+{
+	[EditorBrowsable(EditorBrowsableState.Never)]
```

Comment:
This probably should not be hidden as it is required now that it is a base interface.